Documentation/compatibility/uri-unicode-bidirectional-characters.md

```diff
@@ -0,0 +1,38 @@
+## Allow Unicode Bidirectional Control Characters in URIs
+
+### Scope
+Minor
+
+### Version Introduced
+4.7.2
+
+### Source Analyzer Status
+NotPlanned
+
+### Change Description
+Unicode specifies several special control characters used to specify the orientation of text.
+In previous versions of the .NET Framework, these characters would be incorrectly stripped
+from all URIs. The characters would be stripped even if they were present in their percent
```

Comment:
more concisely: "from all URIs even if they were present in their percent-encoded form."